src/clj_github/repository.clj

```diff
@@ -195,3 +198,23 @@
     (fetch-body! client {:path   url
                          :method :put
                          :body   params})))
+
+(defn clone
+  "Download github repository and put content on destination path"
+  ([client org repo dest]
+   (clone client org repo "master" dest))
+  ([client org repo tag dest]
+   (let [clone-path "/tmp/clone-repo/"
+         url (format "/repos/%/%/zipball/%" org repo tag)
+         git-response (client/request client {:path   url
+                                              :method :get
+                                              :as     :byte-array})
+         filename (->> git-response :headers :content-disposition (re-find #"filename=(.*).zip") second)]
+     (fs/mkdir clone-path)
+     (-> git-response
+         :body
+         (io/input-stream)
+         (clojure.java.io/copy (clojure.java.io/file (str clone-path "git-response.zip"))))
```

Comment:
`clojure.java.io/file` can receive two arguments: the parent folder and the name of the file, then you don't need to use `str`. I believe you could do the same below.